reservation_units/models.py

```diff
@@ -35,6 +35,18 @@ class ReservationUnit(models.Model):
     def __str__(self):
         return "{}".format(self.name)
 
+    def get_location(self):
+        # For now we assume that if reservation has multiple spaces they all have same location
+        spaces = self.spaces.all()
+        if len(spaces) > 0 and hasattr(spaces[0], "location"):
```

Comment:
Is it safe to assume if one (first)space has not location, then none of them have? Or would it be safer to do something like.

`return next((obj for obj in spaces if hasattr(obj, "location")), None)`

Would get rid of the hard coded index too which is in my opinion a bit ugly.